### DISCUSSION TITLE
ScriptReference/Screen.GetDisplayLayout

### DISCUSSION THREAD

**Discussion by giscus** [Developer-Notes-Extension/Unity-Notes#674] [Notes]
# ScriptReference/Screen.GetDisplayLayout

https://docs.unity3d.com/ScriptReference/Screen.GetDisplayLayout.html

**Comment by mrcarriere** [comment]
Testing on an **M1 Macbook Pro** running **macOS 14.5 Sonoma**, it would seem that the list is filled out in order of priority. 

As in, the 0th element in the list is safe to assume as the primary/main display.
